src/PointwiseFunctions/GeneralRelativity/Ricci.cpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/GeneralRelativity/Ricci.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+tnsr::aa<DataType, SpatialDim, Frame, Index> compute_ricci(
```

Comment:
compute_ricci -> ricci_tensor

Reply:
Done. And compute_christoffel_1st_kind -> christoffel_1st_kind as well.